### PR TITLE
DS-2963, DS-3301, DS-3398 Updating VPN SaaSboard to hide fraudulent subs, failed payment records, and FPN 6month free subs

### PR DESCRIPTION
**Pull request by YeonjooSmith**
Related tickets:

- https://mozilla-hub.atlassian.net/browse/DS-2963
- https://mozilla-hub.atlassian.net/browse/DS-3303
- https://mozilla-hub.atlassian.net/browse/DS-3398

All changes except the change in subscriptions.view file were done through the GUI so **the LookML codes were autogenerated.** 

Checklist for reviewer:

When adding a new derived dataset:
- [ ] Ensure that the data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data may be available in [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).
- [ ] Avoid merging a PR that includes the logic of a [core metric](https://docs.telemetry.mozilla.org/metrics/index.html) or complex business logic. The recommendation is to implement core business logic in bigquery-etl. E.g. The [type of search](https://github.com/mozilla/bigquery-etl/blob/a3e59f90326816a2ecaaa3e9d5b57fe9552f7d70/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/query.sql#L781) or the [calculation of DAU or visited URIs](https://github.com/mozilla/bigquery-etl/blob/9bca48821a8a0d40b1700cc14ecd8068d132ed06/sql/moz-fx-data-shared-prod/telemetry_derived/firefox_desktop_exact_mau28_by_dimensions_v1/query.sql).
- [ ] Avoid merging code in Looker Explores/Views that implement analysis with multiple lines of code or that will be likely replicated in the future. Instead, aim for extending an existing dataset to include the required logic, and use [Looker aggregates](https://cloud.google.com/looker/docs/aggregate_awareness) to facilitate the analysis.
- [ ] Avoid merging a PR with logic that requires validation and health checks. It is recommended to implement it in bigquery-etl for full test coverage and failure alerts.

### CODE REVIEW

File: mozilla_vpn/dashboards/vpn_saasboard_retention.dashboard.lookml [hunk 1]
```diff
@@ -1264,143 +1493,6 @@
...
-      Pricing Plan: subscriptions.pricing_plan
-      Provider: subscriptions.provider
-      Plan Interval Type: subscriptions.plan_interval_type
-      Product Name: subscriptions.product_name
-      Original Subscription Start Date: subscriptions.original_subscription_start_date
-    row: 58
-    col: 0
-    width: 24
-    height: 7
   filters:
```
Comment (sean-rose) [inline mozilla_vpn/dashboards/vpn_saasboard_retention.dashboard.lookml:1496]: It doesn't look like any filters for plan ID or fraudulent charges were added to this retention dashboard.  Is that intentional?

**Comment by sean-rose** [comment]
> Note: This is part1 because I will have to abandon this branch after this changes are merged to the main due to some git branch conflict problem.

I think I resolved the branch conflict, so do you want to expand this PR to encompass all the planned changes instead of just part 1?

File: mozilla_vpn/dashboards/vpn_saasboard_retention.dashboard.lookml (hunk 1 above)
Comment (YeonjooSmith) [reply mozilla_vpn/dashboards/vpn_saasboard_retention.dashboard.lookml:1496]: No it is not.. Let me check if they got lost during my battle with git.

Comment (YeonjooSmith) [reply mozilla_vpn/dashboards/vpn_saasboard_retention.dashboard.lookml:1529]: The default values for the filters start from line#1627

**Comment by YeonjooSmith** [comment]
> want to

**Review by sean-rose — APPROVED** [review]

**Pull request merged**